spec/Section 5 -- Validation.md

```diff
@@ -570,6 +577,38 @@ fragment conflictingDifferingResponses on Pet {
 }
 ```
 
+Fragment spread arguments can also cause fields to fail to merge.
+
+```graphql example
+fragment commandFragment($command: DogCommand!) on Dog {
+  doesKnowCommand(dogCommand: $command)
+}
+
+fragment potentiallyConflictingArguments(
+  $commandOne: DogCommand!
+  $commandTwo: DogCommand!
+) on Dog {
+  ...commandFragment(command: $commandOne)
+  ...commandFragment(command: $commandTwo)
+}
+
+query {
+  pet {
+    ...potentiallyConflictingArguments(commandOne: SIT, commandTwo: DOWN)
+  }
+}
+```
+
+If two fragment spreads with the same name, and hence the same selection, supply
+different argument values, their fields will not be able to merge. In this case,
+validation fails because the fragment spread `...commandFragment(command: SIT)`
+and `...commandFragment(command: DOWN)` are part of the visited selections that
+will be merged.
+
+If both of these spreads would have `$commandOne` or `$commandTwo` as the
+argument-value, it would be allowed as we can be sure that we'd resolve
+identical fields.
```

Comment:
`or` here would not be interpreted as an exclusive or. Let's be more explicit:

```suggestion
If both of these spreads had used `$commandOne` as the argument value, it would
be allowed as we can be sure that we'd resolve identical fields.
```

Though of course it would not be valid since then `potentiallyConflictingArguments` would define an argument that wasn't used. Instead, maybe add an example that is valid?

Reply:
Another way to potentially make this valid is to call it as `...potentiallyConflictingArguments(commandOne: SIT, commandTwo: SIT)` or even `...potentiallyConflictingArguments(commandOne: $dogCommand, commandTwo: $dogCommand)` where `$dogCommand` is an operation variable.